package.json

```diff
@@ -1,8 +1,24 @@
 {
   "main": "font",
+  "author": [
+    "LukeXuan"
```

Comment:
Modify to all contributors listed in github and generic authors appended. It was my fault to add only my name on it when I want to complete the `package.json`. Please forgive my offensive.